assets/styles.scss

```diff
@@ -46,6 +46,14 @@ $colour-notification-banner: #ffc107;
   }
 }
 
+// Only show logo on desktop.
+.home-page-logo {
+    display: none;
+    @include breakpoint(break-3) { // 1024px  ~ Desktop up
```

Comment:
Maybe `break-2` is better? I was initially expecting the breakpoint to be the same as the one where the navigation bar collapses to the side.